src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/FunctionProbes/FunctionProbesScenario.cs

```diff
@@ -252,6 +257,17 @@ private static async Task Test_RequestInstallationOnProbeFunctionAsync(FunctionP
             await Assert.ThrowsAnyAsync<OperationCanceledException>(async () => await WaitForProbeInstallationAsync(probeManager, probeProxy, new[] { method }, timeoutSource.Token));
         }
 
+        private static async Task Test_AssertsInProbesAreCaughtAsync(FunctionProbesManager probeManager, PerFunctionProbeProxy probeProxy, CancellationToken token)
+        {
+            MethodInfo method = typeof(SampleNestedStruct).GetMethod(nameof(SampleNestedStruct.DoWork));
+            SampleNestedStruct nestedStruct = new();
+
+            await Assert.ThrowsAnyAsync<XunitException>(async () => await RunInstanceMethodTestCaseAsync(probeManager, probeProxy, method, new object[]
```

Comment:
Can you add a comment as to why this should be expected to fail? I think it's because you are saying it has a `this` parameter, but in reality it does not, so the `Assert.Equal(thisParameterSupported ? thisObj : null, actualArgs[0]);` fails.